Sources/Rownd/Rownd.swift

```diff
@@ -222,6 +222,24 @@ public class Rownd: NSObject {
         }
     }
 
+    public static func signOut(with: RowndSignoutScope) {
+        switch with {
+        case .all:
+            Task {
+                do {
+                    let signOutRequest = SignOutRequest(signoutAll: true)
+                    try await Auth.signOutUser(signOutRequest: signOutRequest)
+                    //sign out of current session
+                    signOut()
+                } catch {
+                    logger.error("Failed to sign out user from all sessions: \(String(describing: error))")
```

Comment:
We should probably `throw` here so that the developer can handle the case where sign-out fails.